src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -43,7 +43,25 @@
  * <pre>
  * &lt;module name=&quot;MissingSwitchDefault&quot;/&gt;
  * </pre>
- *
+ * <p>Example: </p>
+ * <pre>
+ *switch (i) {
+ *  case 1:
+ *    break;
+ *  case 2:
+ *    break;
+ *  // violation
+ *}
+ * </pre>
+ * <p>Example: </p>
```

Comment:
Please make this different in both sections, e.g.  "Example of violation" above and "Example of correct code" here

Reply:
Done